R/JHU.R

```diff
@@ -0,0 +1,163 @@
+#' R6 Class containing specific attributes and methods for John Hopkins
+#' University data
+#'
+#' @description Country specific information for downloading, cleaning
+#'  and processing covid-19 region data for John Hopkins University.
+# nolint start
+#' @source \url{https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data}
+# nolint end
+#' @export
+#' @concept dataset
+#' @examples
+#' \dontrun{
+#' national <- JHU$new(verbose = TRUE, steps = TRUE, get = TRUE)
+#' national$return()
+#' }
+JHU <- R6::R6Class("JHU", # rename to country name
+  inherit = CountryDataClass,
+  public = list(
+
+    # Core Attributes (amend each paramater for country specific infomation)
+    #' @field origin name of country to fetch data for
+    origin = "John Hopkins University (JHU)",
+    #' @field supported_levels A list of supported levels.
+    supported_levels = list("1", "2"),
+    #' @field supported_region_names A list of region names in order of level.
+    supported_region_names = list("1" = "country", "2" = "region"),
```

Comment:
do they have level 3 as well? If yes we should support.

Reply:
No just Country/Region and Province/State in the raw data